build.gradle

```diff
@@ -3,6 +3,8 @@ plugins {
   id 'maven-publish'
   id 'signing'
   id 'net.researchgate.release' version '2.6.0'
+  id 'io.codearte.nexus-staging' version '0.21.2'
```

Comment:
Apparently this plugin required running the task `closeAndReleaseRepository` and I suspect you're planning to add it on the CircleCI release job.

If that's the case, please consider testing to to avoid the need to update that workflow and instead attach this plugin into the `release` task (similarly to the maven-publish): `closeAndReleaseRepository.mustRunAfter release` or `closeAndReleaseRepository.mustRunAfter publish`

Reply:
Good catch! My local testing was using an explicit `./gradlew publish closeAndReleaseRepository ...` and I forgot to add it to the workflow.

I'm not sure how the magic worked that allow the `closeAndReleaseRepository` to execute correctly in the midst of the `release` operations, but I'm willing to try it with a `mustRunAfter` instead. I'm gonna take a wild guess, and try `closeAndReleaseRepository.mustRunAfter publish` first. May take some experimenting on master to test it. YOLO.